pkg/cmd/kafka/consumergroup/describe/describe.go

```diff
@@ -166,6 +166,11 @@ func mapConsumerGroupDescribeToTableFormat(consumers []kafkainstanceclient.Consu
 			CurrentOffset: int(consumer.GetOffset()),
 			OffsetLag:     int(consumer.GetLag()),
 		}
+
+		if consumer.GetMemberId() == "" {
+			row.MemberID = "unconsumed"
```

Comment:
When displayed in the table it does not stand out enough - users might think this is what the actual member ID is.

I think we should do something to highlight it, perhaps by changing the color of the text and capitalising it. Or something along those lines?

Reply:
Is it possible to italicize it?

Reply:
Yes it is. I suggest play around with some different formats and see what fits the best.

Reply:
![Screenshot from 2021-07-28 13-42-14](https://user-images.githubusercontent.com/23582438/127290232-8bee9d69-32dc-40a4-a1de-fb02ac319fde.png)
![Screenshot from 2021-07-28 13-42-37](https://user-images.githubusercontent.com/23582438/127290272-560e71e0-66cd-46fd-962e-5166f6bbd8e7.png)
![Screenshot from 2021-07-28 13-43-04](https://user-images.githubusercontent.com/23582438/127290347-ee26214e-7306-406b-9c87-12fef2c947db.png)
![Screenshot from 2021-07-28 13-44-03](https://user-images.githubusercontent.com/23582438/127290369-9029b9e1-fd1b-4e25-bfee-5f5a78b21562.png)
![Screenshot from 2021-07-28 13-56-49](https://user-images.githubusercontent.com/23582438/127290396-34c2a21c-7201-4d90-b386-7dfe902db25f.png)

Tried a few variations, italics don't seem to be very much distinct. I guess bold and `info` will be better, wdyt?

Reply:
Upon seeing these colours I don't think any of them suit in this setting. How about yellow?

![image](https://user-images.githubusercontent.com/11743717/127314464-bea9cf0f-4242-4650-b7ba-d0d5f76b211b.png)

Reply:
How does the UI represent this?

Reply:
UI is supposed to represent it as small-cased italics. 
_unconsumed_

Reply:
UI doesn't display a count of unconsumed partitions with a sub-heading though. I feel it should be there, so I stuck to that in CLI. Wdyt?

Reply:
Yeah I think it is okay to have it in the CLI as the UI and CLI do not need to exactly match - just as long as the minimum required data is showed.